drivers/virtio/mmio/virtio_mmio.c

```diff
@@ -133,40 +138,40 @@ static int vm_get(struct virtio_dev *vdev, __u16 offset,
 	__u16 w;
 	__u32 l;
 
-	if (vm_dev->version == 1) {
+	if (vm_dev->version <= 2) {
```

Comment:
```suggestion
	if (vm_dev->version == 1) {
```

When I did this hacky change, the default case branch from the switch-case was not there. Now that we have it, there is no need for having such hack.

FTR, the default case branch is needed since the MAC address has 6 bytes and there is no way apply what the virtio standard defines (4.2.2.2 Driver Requirements: MMIO Device Register Layout)

Reply:
Yes you're right, thanks for spotting that 👍🏼